service/src/test/java/uk/gov/hmcts/reform/fpl/service/docmosis/DocmosisCoverDocumentsServiceTest.java

```diff
@@ -37,16 +40,18 @@
     @Autowired
     private DocmosisCoverDocumentsService underTest;
 
-    @BeforeEach
-    void setup() {
-        given(documentGeneratorService.generateDocmosisDocument(any(DocmosisData.class), any()))
-            .willReturn(docmosisDocument);
-    }
 
     @Test
     void shouldGenerateExpectedDocumentWhenAllDataProvided() {
-        DocmosisDocument pdfDocument = underTest.createCoverDocuments(FAMILY_MAN_NUMBER,
-            CCD_CASE_NUMBER, testRepresentative, Language.ENGLISH);
+        given(documentGeneratorService.generateDocmosisDocument(any(DocmosisData.class), eq(COVER_DOCS),
+            eq(RenderFormat.PDF), eq(Language.ENGLISH))).willReturn(docmosisDocument);
```

Comment:
might be worthwhile changing `eq(Language.ENGLISH)` to `eq(LANGUAGE)` so that it is clearer that it is using the language passed rather than giving the impression that no matter what is passed the language here will be `ENGLISH`